src/main/java/abr/user_login_abr/UserLogResponseModel.java

```diff
@@ -13,7 +14,8 @@ public class UserLogResponseModel {
     String passWord;
     BufferedImage userAvatar;
     LocalDateTime accountCreateTime;
-    HashMap<String, String> friendList = new HashMap<>();
```

Comment:
Playlist impl looks good, could you explain why user login response model needs friend list attribute?

Reply:
Every time the User login, The user login Usecase will automatically update the current login User's information into UseStatusViewModel. So that other UI can look up into the ViewModel to check for the Current login User status. And this includes a friend list and all other attributes(Like Username and Playlist)